backend/src/services/secret-v2-bridge/secret-v2-bridge-service.ts

```diff
@@ -409,6 +499,15 @@ export const secretV2BridgeServiceFactory = ({
           })
     });
     if (!secretToDelete) throw new NotFoundError({ message: "Secret not found" });
+    ForbiddenError.from(permission).throwUnlessCan(
+      ProjectPermissionActions.Create,
```

Comment:
I think this should be `ProjectPermissionActions.Delete`

Reply:
Good catch